public_html/wp-content/plugins/pattern-directory/includes/pattern-post-type.php

```diff
@@ -5,6 +5,7 @@
 use Error, WP_Block_Type_Registry;
 use function WordPressdotorg\Locales\{ get_locales, get_locales_with_english_names, get_locales_with_native_names };
 use function WordPressdotorg\Pattern_Directory\Favorite\get_favorite_count;
+use const WordPressdotorg\Pattern_Directory\Pattern_Flag_Post_Type\TAX_TYPE;
```

Comment:
Could this be renamed on import so it's easier to remember which taxonomy it is?

Reply:
Sure, changed it to `FLAG_REASON` in 3ce923e